packages/orchestration/scripts/fetch-chain-info.ts

```diff
@@ -12,7 +12,7 @@ const outputFile = 'src/fetched-chain-info.js';
 /**
  * Names for which to fetch info
  */
-const chainNames = [
+export const icaChainNames = new Set([
```

Comment:
Thanks for including this distinction.

I would suggest keeping a single list here and moving `icaChains` to `chain-capabilities.js` as `IcaEnabled`.

It's a bit of duplication, but in #10329 it was agreed it was best to keep non-chain registry info out of fetch-chain-info / fetched-chain-info.

Reply:
I forgot about that. Thanks for catching it @0xpatrickdev .

Reply:
Done